src/Text/Pandoc/Writers/ODT.hs

```diff
@@ -87,7 +87,7 @@ pandocToODT opts doc@(Pandoc meta _) = do
                                 P.readDataFile "reference.odt"
   -- handle formulas and pictures
   -- picEntriesRef <- P.newIORef ([] :: [Entry])
-  doc' <- walkM (transformPicMath opts) $ walk fixDisplayMath doc
+  doc' <- walkM (transformPicMath opts) $ walk wrapMath $ walk fixDisplayMath doc
```

Comment:
We should think about whether some of these transformations should go in the OpenDocument writer rather than here.  (We had to put tarnsformPicMath here, originally at least, because OpenDocument writer was pure. Now that we have PandocMonad we could even think about putting it all in the OpenDocument writer.)

Reply:
I removed `wrapMath`. It was not needed anymore.